docs/contribute/release_checklist.md

```diff
@@ -20,6 +20,14 @@ permalink: /contributing/release-checklist
 - Wait for taskcluster to complete successfully
 - Check that new versions are on [maven.mozilla.org](https://maven.mozilla.org/?prefix=maven2/org/mozilla/components/):
 
+### Minor releases
+
+Making a minor release has a couple extra steps:
+
+- Create a branch with the format `releases/[major_version].0`, for example, `releases/49.0`.
+- [Cherry-pick](https://git-scm.com/docs/git-cherry-pick) the changes to the new branch.
+- Create the tag and release from this branch instead.
```

Comment:
Let's also mention bumping the version in `.buildconfig` here. That's what I forgot for dot releases a couple of times... :)

Reply:
Good catch, I forgot to add that to the doc as well. 😄